app/scripts/modules/kubernetes/loadBalancer/configure/wizard/upsert.controller.js

```diff
@@ -148,11 +148,11 @@ module.exports = angular.module('spinnaker.loadBalancer.kubernetes.create.contro
     this.submit = function () {
       var descriptor = isNew ? 'Create' : 'Update';
 
+      this.updateName();
```

Comment:
Before this change, `loadBalancerName` would be set on the call as a result of it being passed in `params`. This attribute isn't needed anymore? Trying to understand the change and it doesn't look like cleanup, but a change in behavior.